packages/react-devtools-shared/src/backend/views/Highlighter/Overlay.js

```diff
@@ -226,9 +227,24 @@ export default class Overlay {
 
     if (!name) {
       name = elements[0].nodeName.toLowerCase();
-      const ownerName = getOwnerDisplayName(elements[0]);
-      if (ownerName) {
-        name += ' (in ' + ownerName + ')';
+
+      const node = elements[0];
+      const hook: DevToolsHook =
+        node.ownerDocument.defaultView.__REACT_DEVTOOLS_GLOBAL_HOOK__;
```

Comment:
😳

Reply:
<not the droids you're looking for meme>